gulpfile.js/tasks/styles.js

```diff
@@ -30,8 +31,9 @@ gulp.task('styles:sass', function () {
         .pipe(sass({
             errLogToConsole: true,
             includePaths: includePaths,
-            outputStyle: 'expanded'
+            outputStyle: 'compressed'
         }))
+        .pipe(cssnano())
```

Comment:
We could look at adding dev/prod ENVs that control when this happens. One common pattern via `gulp-util` is this:

```js
...
.pipe(process.env.NODE_ENV === 'production' ? cssnano() : gutil.noop())
...
```